src/CMakeLists.txt

```diff
@@ -133,6 +133,9 @@ add_library(libavrdude
     confwin.c
     crc16.c
     crc16.h
+    developer_opts.c
```

Comment:
I wonder whether we want the developer opts files be part of the library or not.
If they are not going to be part of the library but only the standard CLI tool (my preference), they need to go into a different section in CMakeLists.txt and Makefile.am.
If they are going to be part of the library, public function declarations ought to be in libavrdude.h rather than in a separate header file.

Reply:
I think it is good for the developer opts files to be part of the library.